### ISSUE TITLE
A11y: Colour contrast and information conveyed exclusively via colour is not accessible

### ISSUE THREAD

**Issue by daniellekirkwood** [element-hq/element-web#24931]
Information conveyed exclusively via colors is not accessible to blind and color-blind users. Information should therefore also be made available through additional means.

A notice: The contrast ratios in the alternative displays "dark" and "high contrast" also do not
meet the minimum requirements. Selected menu elements (example marked in red) of the navigation areas are contrasted too little with a ratio of 1.2:1 and 1.4:1 compared to non-selected menu elements (example marked in blue). The minimum requirement of 3:1 is therefore not met.

<img width="129" alt="Screenshot 2023-03-24 at 13 11 29" src="https://user-images.githubusercontent.com/89144281/227530148-36f0af39-2b9e-43a9-b217-88b80d14d746.png">

**Comment by t3chguy** [comment]
Worth pointing out this does not affect screen reader users as we expose the information also via `aria-selected`

<img width="436" alt="image" src="https://user-images.githubusercontent.com/2403652/227533176-703f5686-5ec4-46dd-bad8-8a88a12efa65.png">

**Comment by t3chguy** [comment]
https://github.com/vector-im/element-web/issues/14858

**Comment by daniellekirkwood** [comment]
@janogarcia  I'm assigning this to you as while we're working on the colour contrast the actual underlying issue here is how and where we use colour to convey important information which should be addressed as part of the compound work.

**Comment by janogarcia** [comment]
This won't be relevant anymore once we implement the new designs @americanrefugee has been working on. I'm making sure we don't rely exclusively on color exclusively to convey information for the new UIs/components.

**Comment by daniellekirkwood** [comment]
Ok great -- we'll leave this open for now but know that it has a dependancy on that other work. 

@janogarcia : Note that while the example shown is the header (soon to be updated) there are likely to be other places in the app that run into these sorts of issues

**Comment by daniellekirkwood** [comment]
Great! Closing :D
